src/app/navigation/utils.ts

```diff
@@ -16,12 +19,14 @@ export const locationChangeForRoute = <R extends AnyRoute>(
 ): locationChange is Required<LocationChange<Match<R>>> =>
   !!locationChange.match && locationChange.match.route.name === route.name;
 
+export const getUrlRegexParams = <T extends {}>(obj: T): T => flatten(obj, {delimiter});
```

Comment:
This one does too,  `pathToRegex` doesn't though [(for generating tokens)](https://github.com/pillarjs/path-to-regexp/blob/7b4078db2cfc929eb287432be0ecd3798b0de820/src/index.ts#L141)